pkgs/test/src/install.ts

```diff
@@ -12,18 +12,25 @@ import Zip from "adm-zip"
 import { Octokit } from "@octokit/rest"
 import { join } from "path"
 import { version } from "../package.json"
+import { extractVersionFromDirString } from "./installHelpers"
 export { version } from "../package.json"
 
-export const install = async (versionDir: string) => {
-    console.log(`Installing Redo (version ${version})...`)
-    ensureDir(versionDir)
+export const getRelease = async (version: any) => {
     const gitHub = new Octokit().rest
     const { data } = await gitHub.repos.getReleaseByTag({
         owner: "re-do",
         repo: "redo",
         tag: `v${version}`
     })
-    const zipName = getRedoZipFileName(getOs(), version)
+    return data
+}
+
+export const install = async (versionDir: string) => {
```

Comment:
Pass version directly as a paremeter from getPath so we don't have to recalculate it. Optional to either pass in versionDir as well or just use fromRedo(version) again.